webapp/javascript/ui/Modals/Modal.module.css

```diff
@@ -2,10 +2,6 @@
   background-color: var(--ps-ui-foreground);
 }
 
-.popup label {
-  color: var(--ps-neutral-2);
-}
-
```

Comment:
I removed this style because it hide inputLabel in modal window and looks like we are not using inputLabel anywhere else